CIP-1694/README.md

```diff
@@ -1339,6 +1315,69 @@ We solve the long-term participation problem by not allowing reward withdrawals
 (after the bootstrap phase) unless the stake is delegated to a DRep
 (including the two special cases, namely 'Abstain' and 'No confidence').
 
+### Changelog
+
+#### Changes post Longmont workshop (March 2023)
+
+* Thank the workshop attendees.
+* We have added Constitutional Committee term limits.
+  The term is renewed each time they successfully elect a new committee.
```

Comment:
This really sounds like there is one term for the whole committee

Reply:
Yeah, that is how it used to be. I'll change it slightly.